tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -52,9 +52,8 @@ def test_it_doesnt_add_report_submission_config_if_required_param_missing(
         assert "submissionParams" not in context.js_config.config
 
     def test_it_doesnt_add_report_submission_config_if_lms_not_canvas(
-        self, context, pyramid_request, lti_outcome_params
+        self, context, pyramid_request
     ):
-        pyramid_request.params.update(lti_outcome_params)
         pyramid_request.params.update(
             {"tool_consumer_info_product_family_code": "whiteboard"}
         )
```

Comment:
This is a little OTT for setting a single value :woman_shrugging:

Reply:
Fixed